superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -140,6 +142,7 @@ export default function transformProps(
       return prev + (value as number);
     }, 0);
     totalStackedValues.push(values);
+    thresholdValues.push((percentageThreshold / 100) * values);
```

Comment:
Since the controls sometimes handle missing/unset values in funny ways, I'd probably add this, just in case:
```js
    thresholdValues.push(((percentageThreshold || 0) / 100) * values);
```

Reply:
Good point! I'll get this added